snap/response.varfile

```diff
@@ -1,4 +1,4 @@
-# install4j response file for ESA SNAP 7.0
```

Comment:
Also here I am not sure - this response file was created with ESA SNAP 7.0 so this might be a hint if something breaks in the future...

Reply:
agreed - reinstated